src/components/index.ts

```diff
@@ -1,3 +1,7 @@
 export * from "./posts/PostList";
 export * from "./layout/Layout";
-
+export * from "./login-register/Login-form";
```

Comment:
We can implement eslint in the future here to set some rules that will help you in the future.